src/qml/LocatorItem.qml

```diff
@@ -0,0 +1,158 @@
+import QtQuick 2.11
+import QtQuick.Controls 2.4 as Controls
+import "js/style.js" as Style
+import org.qgis 1.0
+import org.qfield 1.0
+
+
+Item {
+  id: locatorItem
+  property LocatorModelSuperBridge locator: LocatorModelSuperBridge {
```

Comment:
Is this something that is supposed to be overwritten like a parameter?
It looks like a mostly internal thing, so might be a child component instead of a property?

Reply:
indeed